sensor_msgs/LaserScan S
float32[9] R
geometry_msgs/Point32 T
---
sensor_msgs/LaserScan S_prime
